### ISSUE TITLE
Could you please provide the aep file in the example code, preferably on the youtube demo video, I will be very grateful!

### ISSUE THREAD

**Issue by youzhajun** [inlife/nexrender#775]
Hello, I am a java programmer, and I lack some knowledge of AE. Could you please provide the aep file in the example code, preferably on the youtube demo video, I will be very grateful!

**Comment by youzhajun** [comment]
Hello, I tried to make an ae template for demo by myself, and there was an error when I replaced the material, either the sound or the music could be replaced. When I replaced the picture material, there was an error, but there was no obvious error prompt, I was very distressed.Can you help me，thanks!

![image](https://user-images.githubusercontent.com/84554002/194827186-f5717573-7768-463a-8cc5-a09c2dbba8fa.png)

**Comment by limzykenneth** [comment]
@youzhajun Are you running on some kind of virtual machine? As it seems to be detecting a device with only 4MB of RAM.

**Comment by youzhajun** [comment]
@limzykenneth Hello, I am running this program on a virtual machine, but I have allocated 8GB of memory to the virtual machine.  when I replaced the material, either the sound or the music could be replaced. When I replaced the picture material, there was an error,I am very confused

**Comment by limzykenneth** [comment]
The 8GB is likely only allocated to be used by the CPU while the GPU portion seems to be only getting 4MB. You can try to change the settings in AE so that it only use CPU and no GPU or you can try to allocate a bit more RAM to the GPU. With such a low amount of RAM for the GPU it is a bit hard to know what AE will do if it tries to use the GPU for something which I guess in your case replacing image assets uses the GPU while audio assets doesn't (which make some sense).

**Comment by youzhajun** [comment]
@limzykenneth Hello, do you know any open source tools that can parse aepx files? I want to know what layers are in the aepx file.I want to convert the aepx file to json

**Comment by limzykenneth** [comment]
AEPX files are pretty hard to parse and fully parsing it would mean rewriting significant part of after effects itself so there isn't much out there. The easier solution to manipulate an AE project is to use a JSX script directly or via [functions](https://github.com/inlife/nexrender#complete-functions-example).

**Comment by youzhajun** [comment]
hello，I want to make a program with a UI interface for users to operate, so I want to parse and render the apex file to the page in advance, so that users can understand the template through ui, replace the material and then render new videos to users through nexrender. I don't know if you have any good suggestions in this regard.
